### ISSUE TITLE
 Your platform is not compatible yet: linux_x64

### ISSUE THREAD

**Issue by hotmaycold** [NectarJS/nectarjs#59]
```
[!] Your platform is not compatible yet: linux_x64
[!] Please open an issue: https://github.com/nectarjs/nectarjs
```

**Comment by adrien-thierry** [comment]
@hotmaycold Linux x64 and arm64 has been added, could you test ?

**Comment by glococo** [comment]
I just did ./nectar-linux-x64-0_2_4 on my Ubuntu 19.10 and got segmentation fault.
kernel: 5.3.0-10-generic #11-Ubuntu SMP

**Comment by adrien-thierry** [comment]
@glococo what is your processor brand / type ?

**Comment by adrien-thierry** [comment]
Corrected v0.2.7
